src/js/components/Calendar/__tests__/__snapshots__/Calendar-test.js.snap

```diff
@@ -139,7 +139,6 @@ exports[`Calendar date 1`] = `
   padding: 0;
   text-align: inherit;
   line-height: 0;
-  padding: 12px;
```

Comment:
The button changes are affecting the padding, and this is why the tests are breaking.
You can see for example how the Calendar buttons are lost their padding 
![Screen Shot 2019-04-16 at 5 12 04 PM](https://user-images.githubusercontent.com/6320236/56249972-00a39b00-606b-11e9-9346-424b689f5ef5.png)

Reply:
Okay ill take a look I saw that it was taking out padding when I ran the tests not sure why yet